components/climber.py

```diff
@@ -1,13 +1,45 @@
+import wpilib
+from rev import CANSparkMax
+from ids import SparkMaxIds
+
+
 class ClimberComponent:
     def __init__(self) -> None:
         self.deployed = False
 
     def deploy(self) -> None:
         self.deployed = True
 
+    def execute(self) -> None:
+        if self.deployed:
+
+            def __init__(self) -> None:
+                self.deployed = False
+
+    def deploy(self) -> None:
+        self.deployed = True
+
     def execute(self) -> None:
         if self.deployed:
             # Deploy the climber
-            pass
+            self.climbingmotor = CANSparkMax(
+                SparkMaxIds.climber, CANSparkMax.MotorType.kBrushless
+            )  # using a Neo with 4:1 4:1 3:1 ratio
+            self.limitswitch = wpilib.LimitSwitch(2)
+
+            def motorCurrent():
+                return CANSparkMax.getOutputCurrent
+
+            TriggeringLevels = 100  # TriggeringLevels Placeholder for the current levels needed to stop
+            limitswitchHit = False  # placeholder for limit switch variable
+            if limitswitchHit == False:
+                self.drive.motor(0.5, 0)
```

Comment:
Which motor are you trying to actuate? is it the chassis or your climb motor

Reply:
I'm trying to actuate the climb motor.  Should it say self.climbingmotor.<something> ?

Reply:
So a general way of thinking is if you want to control an object, u have the object.method()

So in this case you're right!